lms/envs/common.py

```diff
@@ -317,7 +317,7 @@
     'ENABLE_MOBILE_SOCIAL_FACEBOOK_FEATURES': False,
 
     # Enable temporary APIs required for xBlocks on Mobile
-    'ENABLE_COURSE_BLOCKS_NAVIGATION_API': False,
+    'ENABLE_COURSE_BLOCKS_NAVIGATION_API': True,
```

Comment:
Ah, I think I forgot to change it back when I realized we were using the new api.